scripts/test-novu-diagnostics.js

```
@@ -0,0 +1,271 @@
+const path = require('path');
+
+console.log('🔍 Starting Novu Configuration Diagnostics...\n');
+
+// Check if we're in development or need to start the Next.js app
+const isDev = process.env.NODE_ENV !== 'production';
+
+if (isDev) {
+  console.log('🚀 Starting Next.js development server for diagnostics...\n');
+
+  // Start Next.js dev server
+  const nextProcess = spawn('npx', ['next', 'dev'], {
```

ESLint

[error] 'nextProcess' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)